utils/checkEnvironmentVariables.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Function checking an object (taken from process.env)
+ * for the mandatory keys to access the coinbase API.
+ */
+
+const checkEnvironmentVariables = (env) => {
+  const errorMessage = ['Environemnt Variable(s) Missing:'];
+  if (!env.apiKey) {
+    errorMessage.push('\n - apiKey');
+  }
+  if (!env.apiSecret) {
+    errorMessage.push('\n - apiSecret');
+  }
+  if (!env.apiPassphrase) {
+    errorMessage.push('\n - apiPassphrase');
+  }
+  if (errorMessage.length>1) {
```

Comment:
Codacy has a fix for the issue: [Operator '>' must be spaced.](https://app.codacy.com/gh/amcalabretta/botbase/pullRequest?prid=7628479)
```suggestion
  if (errorMessage.length > 1) {
```